flang/test/Intrinsics/sizeof.f90

```diff
@@ -0,0 +1,10 @@
+! RUN: %python %S/test_errors.py %s %flang_fc1
+  character(len=20) :: a, b
+  if (sizeof(a) == sizeof(x=b)) then
+    print *, "pass"
+  else
+    print *, "fail"
+  end if
+  ERROR! unknown keyword argument to intrinsic 'sizeof'
```

Comment:
This file needs to be in flang/test/Semantics in order to use test_errors.py, and it needs to have a syntax error cleaned up.